Marine armory expansion

thenoob699 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Democracy has come in and SOM has allowed Terra Gov the use of their armory.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Adds previously missing SOM/PMC weaponry and unseen TGMC ammunition to the Marine Automated Weapons Rack and updates Valhalla vendors to reflect shipside vendors.

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
SOM
add: V-11 Extended pistol magazine
add: V-11 Incendiary pistol magazine
add: V-21 AP SMG magazine
add: V-21 Incendiary SMG magazine
add: V-21 Extended SMG magazine
add: V-31 AP magazine
add: V-31 Incendiary magazine
add: V-31 Airburst Microgrenades
add: V-31 Dragon's Breath Microgrenades
add: V-31 Clustermunition Microgrenades
add: V-31 Smoke burst Microgrenades 
add: SOM-V51B Assault Shotgun
add: V-71 RPG backpack
Renames the high-explosive RPG warhead to be high-explosive
Renames the SOM flamethrower tank to V-62 large flamerthrower tank
![Armory 1](https://user-images.githubusercontent.com/100187467/208483972-979ef679-69fb-4896-a242-9ae549ce1fe9.png)

PMC
add: PR-412E battle rifle
add: PR-412 Extended magazine
add: PR-412 Incendiary magazine
add: PR-412 AP magazine
add: SMG-25B
add: SMG-25 Extended magazine
add: SMG-25 AP magazine
![Armory 2](https://user-images.githubusercontent.com/100187467/208484011-55d5a69e-6eef-48dc-89b4-7afa0ea8ae3e.png)

Marine
adds: RT-3 (SO Pistol)
adds: RT-3 Hollowpoint magazine
adds: RT-3 AP magazine
adds: RT-3 Incendiary magazine
adds: RT-3 Extended magazine
adds: P-1911 (TWO WORLD WARS)
adds: Adds P-22 (glawk 40)
adds: TL-127 Flak magazines to Automated Weapons Vendor
adds: DMR-37 Incendiary magazines to Automated Weapons Vendor
adds: BR-64 Incendiary magazines to Automated Weapons Vendor
balance: Allows taser to use lasgun and volkite mags
balance: SP-13 magazine capacity from 25 to 50
balance: SP-13 the ability to be fired one handed
balance: DMR-37 Incendiary magazine capacity from 15 to 20
balance: allows DMR-37 the ability to use incendiary mags
balance: gives DMR-37 Incendiary ammo counter the correct incendiary HUD element
![Armory 3](https://user-images.githubusercontent.com/100187467/208484044-04c64d33-7dbb-4c38-9d07-0ed4c116bb83.png)

adds: 10x24mm AP boxes
adds: 10x24mm AP box sprites
adds: 10x20mm AP packets
adds: 10x20mm AP packet sprites
balance: Gives 10x24mm AP ammo box and 10x20mm AP ammo packet able to be loaded into the T-72 Pointman longevity belt
adds: SOM weaponry to Valhalla vendors
Renames TL-127 Flak Magazine in requisitions menu to SR-127 Flak Magaine

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

JasmineRickards commented:
Very good change idea and concept, but done a bit excessively. PMC flak should probably be in req rather than round start available. That is the same with AP rounds and incends. Maybe if you added ammo boxes for that ammo type and magazines into req, this would be more viable. As it stands, it's just a little excessive to add right in.

thenoob699 commented:
I added a new commit but it was a few commits behind, I updated the branch with the new commits  and I hope it doesn't break anything. I'm new to using Git.

thenoob699 commented:
It does not, if you are working on a branch that is behind the master, it will not automatically add your old branch into the main one.
